client/src/components/books/view/BorrowBook.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+/* eslint-disable require-jsdoc, class-methods-use-this */
+class BorrowBook extends React.Component {
+  constructor(props) {
+    super(props);
+    const { userId, bookId } = this.props;
+    this.state = {
+      isLoading: false,
+      userId,
```

Comment:
You should either use the object shorthand `{ key1, key2, key3 }` or the long form `{ key1: val1, key2: val2}`

Reply:
I used the object shorthand. Hence, the object destructuring in line 8